openfoam-interface/internal-interface/tests/test_foam_interface.py

```diff
@@ -0,0 +1,312 @@
+""" test foam native interface
```

Comment:
This test is now for IO wrapper interface. Should be changed to use internal interface and add test of methods which are missing